src/celldega/nbhd/__init__.py

```diff
@@ -300,4 +306,656 @@ def create_hextile(radius, path_landscape_files=None, img_height=100, img_width=
         plt.show()
         plt.close()
 
-    return gdf_hextile
\ No newline at end of file
+    return gdf_hextile
+
+
+def calc_nbg_cd(
+    adata,
+    gdf_nbhd: gpd.GeoDataFrame,
+    cd_mode: str = 'CD/LCD',
+    unique_nbhd_col: str = "name"
+) -> gpd.GeoDataFrame:
+    """
+    Calculate mean gene expression per neighborhood (CD) or per cluster-neighborhood (LCD).
```

Comment:
We can phrase this like "the mean expression of cells within a neighborhood (CD) or the mean expression of cells from a given Leiden cluster (LCD)"